tests/unit/ListTable.test.js

```diff
@@ -64,11 +64,11 @@ describe('when rendering a list-table directive', () => {
     });
 
     it('applies the class passed in as an option', () => {
-        expect(wrapper.find('.guide-tablenate')).toHaveLength(1);
+        expect(wrapper.find('.guide-tablenate')).toHaveLength(2);
```

Comment:
Not sure what was going on here? I looked a bit into recent changes to this component but decided to just update these values to pass and move in, since the component seems to work as expected right now.